docs/index.asciidoc

```diff
@@ -27,11 +27,25 @@ You can use this plugin to create consistent document ids when  events are
 inserted into Elasticsearch. This approach means that existing documents can be
 updated instead of creating new documents.
 
-NOTE: When the `target` option is set to `UUID` the result won't be
+NOTE: When the `method` option is set to `UUID` the result won't be
 a consistent hash but a random
 https://en.wikipedia.org/wiki/Universally_unique_identifier[UUID].
 To generate UUIDs, prefer the {logstash-ref}/plugins-filters-uuid.html[uuid filter].
 
+[id="plugins-{type}s-{plugin}-ecs_metadata"]
+==== Event Metadata and the Elastic Common Schema (ECS)
+This plugin adds a hash value to event as an identity. User can config `target` to change the output field
```

Comment:
```suggestion
This plugin adds a hash value to event as an identifier. You can configure the `target` option to change the output field.

```

Reply:
Did you mean "identifier?"  
Note that I added a return after this sentence to force the paragraph to break for readability.